backend/hubs/models/hub.py

```diff
@@ -300,3 +304,66 @@ def __str__(self):
             self.user.first_name + " " + self.user.last_name,
             self.title,
         )
+    
+class HubSupporter(models.Model):
+    name = models.CharField(
+        help_text="Supporter name",
+        verbose_name="Supporter name",
+        max_length=1024,
+        null=True,
+        blank=True,
+    )
+    subtitle = models.CharField(
+        help_text="Supporter subtitle",
+        verbose_name="subtitle",
+        max_length=1024,
+        null=True,
+        blank=True,
+    )
+    logo = models.ImageField(
+        help_text="Supporter logo",
+        verbose_name="Logo",
+        null=True,
+        blank=True,
+        upload_to=hub_supporter_logo_path,
+    )
+    hub = models.ForeignKey(
+        Hub,
+        help_text="Supported Hubs by the Supporter",
+        verbose_name="Hub",
+        related_name="supporter_hub",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
+    )
+    importance = models.PositiveSmallIntegerField(
+        help_text="The larger the number, the more to the top this hub will be displayed on the hubs overview page",
```

Comment:
I didn't get your point. can you explain what should I do here?
Thanks

Reply:
For example the api that lists all projects will only list projects with importance > 0. This way we can hide a project that requires a change without having to delete it. This could also be quite helpful for supporters.